Makefile.am

```diff
@@ -151,6 +151,7 @@ check_LTLIBRARIES = \
 
 
 check_PROGRAMS = \
+    test_utils_cache \
```

Comment:
Small issue with tab sizes. You can run the format script to fix it 😊

Reply:
I fixed it! Thank you!